webui/react/src/pages/DesignKit.tsx

```diff
@@ -1263,6 +1265,26 @@ const TooltipsSection: React.FC = () => {
   );
 };
 
+const EmptySection: React.FC = () => {
+  return (
+    <ComponentSection id="Empty" title="Empty">
+      <Card>
+        <p>
+          An <code>{'<Empty>'}</code> component indicates that no content is available for a page.
+          It may display an icon and a description explaining why this state is displayed.
+        </p>
+      </Card>
+      <Card title="Usage">
+        <Empty
+          description={<strong>Empty description</strong>}
```

Comment:
nit: Since we're not using `<strong>` in any of the descriptions in the app it seems misleading to use it here.

Reply:
thanks, updated that and also made some more changes if you want to take a look (removing antd Empty usage, changing the icon in a couple places, and changing css to use flexbox to vertically center instead of a margin-top)